superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js

```diff
@@ -100,3 +100,42 @@ describe('Dashboard edit mode', () => {
     });
   });
 });
+
+describe('Single component tests', () => {
+  before(() => {});
+  beforeEach(() => {
+    cy.login();
+    cy.visit('dashboard/new');
```

Comment:
This will create a new dashboard every time this test is run. Which is not very nice when debugging.

Reply:
I placed this to have always initial state to avoid flakiness between test runs. Why is it not good? I see only issue in execution time.